src/averageAges.js

```diff
@@ -55,7 +62,32 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const childrenWithMoms = people.map(el => {
```

Comment:
https://github.com/mate-academy/js_average-ages/pull/1346#discussion_r984759902

check this comment please)
That's an example of a shorter way to write the same code :arrow_up:

Reply:
but no need `newObj.` on each line there

Reply:
@TarasHoliuk thank you so much!!!
I didn't understand girls comments at all.